src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -441,9 +441,28 @@
   "skyblocker.config.mining.crystalsWaypoints": "Crystal Hollows Waypoints",
   "skyblocker.config.mining.crystalsWaypoints.enabled": "Enabled Waypoints",
   "skyblocker.config.mining.crystalsWaypoints.enabled.@Tooltip": "Show a waypoint (waypoint selected in general/waypoints) at important areas in the crystal hollows e.g. Jungle Temple and Fairy Grotto. ",
+  "skyblocker.config.mining.crystalsWaypoints.textScale": "Text Scale",
+  "skyblocker.config.mining.crystalsWaypoints.textScale.@Tooltip": "Scale the size of the commission labels.",
   "skyblocker.config.mining.crystalsWaypoints.findInChat": "Find Waypoints In Chat",
   "skyblocker.config.mining.crystalsWaypoints.findInChat.@Tooltip": "When in crystal hollows read the chat to see if coordinates are sent and extract these to show as waypoint or on the map",
+  "skyblocker.config.mining.crystalsWaypoints.addedWaypoint": "Added waypoint for ",
+  "skyblocker.config.mining.crystalsWaypoints.addedWaypoint.at": "at",
+  "skyblocker.config.mining.crystalsWaypoints.noActive": "You have no active waypoints to share or remove.",
+  "skyblocker.config.mining.crystalsWaypoints.allActive": "You have no more waypoints left to create.",
   "skyblocker.config.mining.crystalsWaypoints.shareFail": "Can only share waypoints you have found.",
+  "skyblocker.config.mining.crystalsWaypoints.removeSuccess": "Removed waypoint for ",
+  "skyblocker.config.mining.crystalsWaypoints.removeFail": "Can only remove waypoints you have found.",
+  "skyblocker.config.mining.crystalsWaypoints.wishingCompassSolver": "Wishing Compass Solver",
+  "skyblocker.config.mining.crystalsWaypoints.wishingCompassSolver.@Tooltip": "Works out and adds a waypoint to where a wishing compass points to",
+  "skyblocker.config.mining.crystalsWaypoints.wishingCompassSolver.enableTabMessage": "Enable crystals in /tab so the compass can know what has been found",
+  "skyblocker.config.mining.crystalsWaypoints.wishingCompassSolver.wishingCompassUsedMessage": "Wishing compass used. Move to another location and use another compass to triangulate target",
+  "skyblocker.config.mining.crystalsWaypoints.wishingCompassSolver.somethingWentWrongMessage": "Something went wrong. lines do not cross. please try again",
```

Comment:
I feel like these sentences could be formatted/worded better, especially the last one.